contrib/scripts/go-mod-update-cloud-providers.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
```

Comment:
nit : Doesn't look like this is needed?

Reply:
It's currently not needed, but most other scripts have this by default and I think it's good practice to already set these in case of future changes introducing piped commands.